File: 250718/특정 조건 두 정수 비교/specific-comparison-of-two-natural-numbers.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();

        // 변수
        int result1 = 0;
        int result2 = 0;

        // 조건문 1
        if (num1 < num2) {
            result1 = 1;
        } else {
            result1 = 0;
        }

        // 조건문 2
        if (num1 == num2) {
            result2 = 1;
        } else {
            result2 = 0;
        }

        // 출력
        System.out.printf("%d %d", result1, result2);
    }
}

File: 250718/특정 조건 두 정수 비교/specific-comparison-of-two-natural-numbers.py
# 입력
num1, num2 = map(int, input().split())

# 조건문 1
if num1 < num2 :
    result1 = 1
else :
    result1 = 0

# 조건문 2
if num1 == num2 :
    result2 = 1
else :
    result2 = 0

# 출력
print(result1, result2, sep = " ")